vars/RedHat_9.yml

```diff
@@ -0,0 +1,41 @@
+---
+
+# vars file for sap-netweaver-preconfigure
+
+__sap_netweaver_preconfigure_sapnotes:
+  - "2526952"
+
+__sap_netweaver_preconfigure_packages:
+  - tuned-profiles-sap
+
+__sap_netweaver_preconfigure_min_swap_space_mb: '20480'
+
+__sap_netweaver_preconfigure_adobe_doc_services_packages:
+  - autoconf.noarch
+  - automake.noarch
+  - cyrus-sasl.x86_64
```

Comment:
do you need to have `noarch`, `x86_64`, etc. in the package names here?

Reply:
This is because it is mentioned like this in SAP note 2920407.